content/en/docs/tasks/configure-pod-container/quality-service-pod.md

```diff
@@ -155,6 +171,9 @@ kubectl delete pod qos-demo-2 --namespace=qos-example
 For a Pod to be given a QoS class of BestEffort, the Containers in the Pod must not
 have any memory or CPU limits or requests.
 
+BestEffort pods have the lowest priority and will be the first pods killed if the system runs
```

Comment:
```suggestion
BestEffort pods have the lowest priority and will be the first pods that get shut down
(through kubelet-initiated {{< glossary_tooltip text="eviction" term_id="eviction" >}})
if the node runs
```

Reply:
This is not quite right, as a node experiencing resource pressure kills, and does not evict nodes.

Reply:
I'm not sure what is truly happening here:
- the kubelet asks the API server to delete a Pod that has breached a resource limit
- the kubelet kills one or more containers belonging to a Pod; specifically, those containers that have breached a resource limit. Regardless of the `.spec` for that Pod, the kubelet does not start a replacement container.
- the kubelet kills one or more containers belonging to a Pod; specifically, those containers that have breached a resource limit. The kubelet start a replacement container if the Pod specifies that behavior.
- the kubelet kills all containers belonging to a Pod and sets a status on the Pod that it has failed / breached a resource limit, then waits for the control plane to notice
- something else

Let's clarify which it is - ideally, in the _concept_ page I'm recommending.